drivers/shared/executor/proto/executor.proto

```diff
@@ -28,9 +28,10 @@ message LaunchRequest {
     string task_dir = 8;
     bool resource_limits = 9;
     bool basic_process_cgroup = 10;
-    repeated hashicorp.nomad.plugins.drivers.proto.Mount mounts = 11;
-    repeated hashicorp.nomad.plugins.drivers.proto.Device devices = 12;
-    hashicorp.nomad.plugins.drivers.proto.NetworkIsolationSpec network_isolation = 13;
+    bool no_pivot_root = 11;
```

Comment:
I would insert append this to the end as item 14.  For backward compatibility, field numbers don't change once they are assigned.